services/types.go

```diff
@@ -50,6 +50,9 @@ type Services interface {
 
 	// PlacementService returns a client of Placement Service
 	PlacementService(service ServiceID, popts PlacementOptions) (PlacementService, error)
+
+	// HeartbeatService returns a heartbeat store for the given zone.
+	HeartbeatService(zone string) (HeartbeatService, error)
```

Comment:
hmm... I actually prefer not to expose a new argument of "zone" here, since all the other methods in this interface pretty much all take in a service id, maybe just do that here as well?